packages/code-du-travail-frontend/src/common/Article.js

```diff
@@ -17,34 +17,40 @@ const Article = ({
     <Section>
       <Container>
         <Wrapper variant="light">
-          <Header>
-            <IconWrapper>
-              <Icon />
-            </IconWrapper>
-            <div>
-              <H1>{title}</H1>
-              <Meta>
-                {sourceType && <Type>{sourceType}</Type>}
-                {date && (
-                  <Date>
-                    Mis à jour le&nbsp;: <DateValue>{date}</DateValue>
-                  </Date>
-                )}
-              </Meta>
-            </div>
-            <Tags>
-              {tags.map(tag => (
-                <Tag
-                  key={tag}
-                  style={{ cursor: "pointer" }}
-                  onClick={() => onTagClick(tag)}
-                >
-                  {tag}
-                </Tag>
-              ))}
-            </Tags>
-          </Header>
-          {children && <Content>{children}</Content>}
+          <Section>
+            <Header narrow noPadding>
```

Comment:
we should add role="header" or as="header" (not sure it works, since it's also a Container)

Reply:
I could add a basic `<header>` tag below and call the container `HeaderContainer` ! Otherwise I could make it work by defaulting it to pass all props to the generated div, but usually I prefer to use the native html element